easybuild/easyconfigs/k/KMCLib/KMCLib-2.0-a2-foss-2023a-Python-2.7.18.eb

```diff
@@ -0,0 +1,51 @@
+easyblock = 'CMakeMake'
+
+name = 'KMCLib'
+version = '2.0-a2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/leetmaa/KMCLib'
+description = """KMCLib - a general framework for lattice kinetic Monte Carlo (KMC) simulations"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+source_urls = ['https://github.com/leetmaa/KMCLib/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['796620a67ad010df4b11734f703151c17441f82cef026f3d56386a34056d0213']
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('CMake', '3.26.3'),
+]
+
+dependencies = [
+    ('Python', '2.7.18'),
+    ('numpy', '1.16.6', versionsuffix),
+    ('SWIG', '4.1.1'),
+]
+
+start_dir = 'c++'
+
+preconfigopts = 'cd ../%(name)s-%(version)s/c++/externals && '
+preconfigopts += 'make CC="$CC" CXX="$CXX" CFLAGS="$CFLAGS" CXXFLAGS="$CXXFLAGS" && cd - && '
+
+prebuildopts = 'export CPATH=$EBROOTPYTHON/include/python%(pyshortver)s:$CPATH && '
+
+postinstallcmds = [
+    'cp -r "%(builddir)s/%(name)s-%(version)s/python/src/KMCLib" "%(installdir)s"',
+    'cp -r "%(builddir)s/../python/src/KMCLib/Backend/"* "%(installdir)s/KMCLib/Backend/"',
+]
+
+modextrapaths = {'PYTHONPATH': ''}
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['KMCLib/Backend'],
+}
+
+sanity_check_commands = [
+    "python -c 'from KMCLib import *'",
+    'python %(builddir)s/%(name)s-%(version)s/python/unittest/utest.py'
```

Comment:
This doesn't agree with `--module-only` or `--sanity-check-only`, since in that case the build directory will not be there...

Maybe we should run this via `test_cmd` instead?